CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+## Contributing
+
+This charm has been built on top of the Apache Bigtop project. Its source code lives in the main Bigtop repository on Github: https://github.com/apache/bigtop
+
+Bug fixes, new features and other contributions are welcome, and encouraged! Since this is a joint project between Canonical Charmers and the Apache Bigtop community, it can be a little tricky to get the code submitted to all the right places, and +1ed by the right people, however.
```

Comment:
I think the steps outlined further down are fairly straightforward and it's not really all that "tricky."  Perhaps we could go more with something like:

> Bug fixes, new features, and other contributions are welcome and encouraged!  Contributing follows the [same process](https://cwiki.apache.org/confluence/display/BIGTOP/How+to+Contribute) as for the rest of the Bigtop project, but can be simplified by using the GitHub integration.  The simplified workflow is as follows: